client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -420,25 +443,26 @@ class EditContactInfoFormCard extends React.Component {
 
 				case WWD:
 				default:
-					const oldEmail = this.props.contactInformation.email,
-						newEmail = formState.getFieldValue( this.state.form, 'email' ),
+					const currentEmails = this.getCurrentEmails(),
 						strong = <strong />;
 
-					if ( oldEmail === newEmail ) {
+					if ( this.hasEmailChanged() ) {
 						message = this.props.translate(
 							'An email has been sent to {{strong}}%(email)s{{/strong}}. ' +
 							'Please confirm it to finish this process.',
 							{
-								args: { email: oldEmail },
+								args: { email: currentEmails },
 								components: { strong }
 							}
 						);
 					} else {
+						const newEmail = formState.getFieldValue( this.state.form, 'email' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Emails have been sent to {{strong}}%(oldEmail)s{{/strong}} and {{strong}}%(newEmail)s{{/strong}}. Please ensure they're both confirmed to finish this process.' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/third-email-whois-update)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).